models/issue.go

```diff
@@ -973,7 +1028,11 @@ func newIssue(ctx context.Context, doer *user_model.User, opts NewIssueOptions)
 	if opts.IsPull {
 		_, err = e.Exec("UPDATE `repository` SET num_pulls = num_pulls + 1 WHERE id = ?", opts.Issue.RepoID)
 	} else {
-		_, err = e.Exec("UPDATE `repository` SET num_issues = num_issues + 1 WHERE id = ?", opts.Issue.RepoID)
+		if opts.Issue.IsPrivate {
+			_, err = e.Exec("UPDATE `repository` SET num_private_issues = num_private_issues + 1 WHERE id = ?", opts.Issue.RepoID)
+		} else {
+			_, err = e.Exec("UPDATE `repository` SET num_issues = num_issues + 1 WHERE id = ?", opts.Issue.RepoID)
```

Comment:
How about `num_issues` includes `num_private_issues`, otherwise we need rename it as `num_public_issues`.

Reply:
I would be in to rename it into `num_public_issues`, as `num_issues` should just act as `num_public_issues`, as otherwise we would need a new column anyway that only has the count of public issues, which then just becomes ineffective.

I'm not sure about how to rename a column with migration, could you help me with that?

Reply:
So I think we can keep `num_issues` compatible as before to store all the issues number, added a new column `num_private_issues` for the new feature.